presto-spi/src/main/java/com/facebook/presto/spi/block/AbstractFixedWidthBlock.java

```diff
@@ -186,6 +186,18 @@ public long getRegionSizeInBytes(int positionOffset, int length)
         return (fixedSize + Byte.BYTES) * (long) length;
     }
 
+    @Override
+    public long getPositionsSizeInBytes(boolean[] positions)
+    {
+        int usedPositionCount = 0;
```

Comment:
I'd add a method to BlockUtil like the Guava's `Booleans.countTrue` https://github.com/google/guava/blob/master/guava/src/com/google/common/primitives/Booleans.java#L513

I also prefer its use of the iterator style for loop.